txpipe/twopoint_null_tests.py

```diff
@@ -302,7 +302,7 @@ class TXGammaTStars(TXTwoPoint):
         "npatch": 5,
         "use_true_shear": False,
         "subtract_mean_shear": False,
-        "use_randoms": True,
+        "use_randoms": False,
```

Comment:
Why should this not be set to true?  Could you add a comment explaining?

Reply:
@jjeffe14 we can flip this back to true, and seeing if it runs through. It might just be me turning it off while testing to make it faster.

Reply:
done!